LibreNMS/service.py

```diff
@@ -605,9 +605,12 @@ def logfile_watchdog(self):
             error("Log file not found! {}".format(e))
             return
 
-        if logfile_mdiff > self.config.poller.frequency:
-            critical("BARK! Log file older than {}s, restarting service!".format(self.config.poller.frequency))
-            self.restart()
+        if self.config.watchdog_enabled:
+            if logfile_mdiff > self.config.poller.frequency:
+                critical("BARK! Log file older than {}s, restarting service!".format(self.config.poller.frequency))
+                self.restart()
+            else:
```

Comment:
The info message in the else does not line up with it does.  (It can only be reached if watchdog is enabled)

Reply:
Does that look better?